### PR TITLE
Feature/day01 cxphoenix

### PR DESCRIPTION
**Pull request by CXPhoenix**
# Use skill
1. Use the event listener to control that user's key down and key up.
2. Use the query selector to play the sound.
3. Use the currentTime property to control the sounds playback.
4. Use the query selector to control the css style.

### CODE REVIEW

File: day01/cxphoenix/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <audio data-key="83" src="sounds/hihat.wav"></audio>
+  <audio data-key="68" src="sounds/kick.wav"></audio>
+  <audio data-key="70" src="sounds/openhat.wav"></audio>
+  <audio data-key="71" src="sounds/boom.wav"></audio>
+  <audio data-key="72" src="sounds/ride.wav"></audio>
+  <audio data-key="74" src="sounds/snare.wav"></audio>
+  <audio data-key="75" src="sounds/tom.wav"></audio>
+  <audio data-key="76" src="sounds/tink.wav"></audio>
+
+<script src="main.js"></script>
```
Comment (kayac-chang) [inline day01/cxphoenix/index.html:60]: use prettier for html format

File: day01/cxphoenix/main.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+window.addEventListener('keydown', e => {
+    const playKey = e.keyCode;
```
Comment (kayac-chang) [inline day01/cxphoenix/main.js:2]: use `key` or `code` instead, please take a look [mdn](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

File: day01/cxphoenix/index.html (hunk 1 above)
Comment (CXPhoenix) [reply day01/cxphoenix/index.html:60]: got it

File: day01/cxphoenix/main.js (hunk 2 above)
Comment (CXPhoenix) [reply day01/cxphoenix/main.js:2]: Ok!

File: day01/cxphoenix/index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  <audio data-key="65" src="sounds/clap.wav"></audio>
+  <audio data-key="83" src="sounds/hihat.wav"></audio>
+  <audio data-key="68" src="sounds/kick.wav"></audio>
+  <audio data-key="70" src="sounds/openhat.wav"></audio>
+  <audio data-key="71" src="sounds/boom.wav"></audio>
+  <audio data-key="72" src="sounds/ride.wav"></audio>
+  <audio data-key="74" src="sounds/snare.wav"></audio>
+  <audio data-key="75" src="sounds/tom.wav"></audio>
+  <audio data-key="76" src="sounds/tink.wav"></audio>
```
Comment (elzuoc) [inline day01/cxphoenix/index.html:58]: please use assets in sample file ~
remember remove these item ~

**Comment by ashramwen** [comment]
pr title needs to be more precise.

**Pull request closed without merging**